stdlib/public/Platform/Platform.swift

```diff
@@ -103,8 +103,7 @@ public var errno : Int32 {
   get {
 #if os(OSX) || os(iOS) || os(watchOS) || os(tvOS) || os(FreeBSD)
     return __error().pointee
-//FIXME: os(Windows) should be replaced, such as triple(Cygwin)
```

Comment:
Yes, I would recommend sending an email to swift-evolution with a subject like "#if os(Windows) and MSVC / Cygwin Compatibility".  It should link to this thread and then briefly summarize it.  CC'ing everybody here is a good idea, too.

For anybody to comes to this PR from swift-evolution: please keep the conceptual discussion on the mailing list.